src/hand.js

```diff
@@ -0,0 +1,37 @@
+export default class Hand {
+
+  constructor(deckId) {
+    this.deckId = deckId;
+    this.element = document.createElement('div');
+    this.element.classList.add('row');
+  }
+
+  getData() {
+    return fetch(`https://card-proxy.herokuapp.com/decks/${this.deckId}/draw?count=5`)
+      .then((res) => res.json())
+      .then((data) => {
+        this.data = data;
+        console.log(data);
+      });
+  }
+  render() {
+    const cardItem = this.data.cards.map((card) => {
+      const cardElement = document.createElement('div');
+      cardElement.classList.add('row');
```

Comment:
Within the `map` you are creating an extra set of rows. Instead this should be a `card` div and then you can remove the `card` from your `innerHTML` below.

This should fix the strange styling that is going on right now:
![screen shot 2016-07-22 at 8 27 28 am](https://cloud.githubusercontent.com/assets/2532004/17058533/7efac02a-4fe6-11e6-951c-78efcba8addd.png)